source/validation/mei-purify.sch

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<sch:schema queryBinding="xslt2" xmlns:sch="http://purl.oclc.org/dsdl/schematron">
+    <sch:ns prefix="tei" uri="http://www.tei-c.org/ns/1.0"/>
+    <sch:ns prefix="rng" uri="http://relaxng.org/ns/structure/1.0"/>
+
+    <sch:pattern id="check_rngChoice">
+        <sch:rule context="rng:choice">
+            <sch:assert role="error" test="count(*) > 1">An &lt;rng:choice&gt; element hast to contain more than one option to choose from.</sch:assert>
+        </sch:rule>
+    </sch:pattern>
+
+    <sch:pattern id="check_teiConstraintSpec_ident">
+        <sch:rule context="tei:constraintSpec">
+            <sch:assert role="error" test="@ident != preceding::tei:constraintSpec/@ident">The @ident on constraintSpec has to be unique across all of mei-source.xml.</sch:assert>
```

Comment:
`mei-source.xml` actually doesn'T contain constraints, better to use something like "across all MEI source files".

Reply:
Hmm… right will work in a context where xi:includes get resolved automatically, e.g. with XInclude processing activated in oXygen XML or when canonicalising first and then validating the canonicalised file like in the newly pushed ant target ;-)

Reply:
I’d be ok with having this as ant-target exclusively; what do you guys think @rettinghaus & @musicEnfanthen?

Reply:
Sounds okay with me.
As it will be of use only for a limited time, maybe add a comment for future us?

Reply:
addd a xml comment